app/TuistApp/Sources/Views/SimulatorRow/SimulatorRowViewModel.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import TuistCore
+import TuistSupport
+
+@Observable
+final class SimulatorRowViewModel {
+    private let simulatorController: SimulatorControlling
+    private let system: Systeming
```

Comment:
I'd use [Command](https://github.com/tuist/command/) here, which is already align with Swift's new concurrency model.

Reply:
I find it a bit confusing for which things we use the new models and for which not. Might be worth writing a guidance for contributors if we're gonna be in this in-between state for a while.